src/peripherals/atecc608a-tnglora-se/atecc608a-tnglora-se.c

```diff
@@ -314,11 +314,13 @@ SecureElementStatus_t SecureElementInit( SecureElementNvmEvent seNvmCtxChanged )
         return SECURE_ELEMENT_ERROR;
     }
 
+    atca_delay_us(1500000);
```

Comment:
The proposed delays comes after the lower level initialization function `atcab_init` and between reads.
This contradicts with the pull request description.

Can you please provide more info on:
- MCU type
- I2C setup (interrupt or polling, frequency,..., etc)
- `atcab_init` invocation return

You can also try increasing `ATCA_HAL_ATECC608A_I2C_WAKEUP_DELAY` value to `3500U` in
 https://github.com/Lora-net/LoRaMac-node/blob/118771fee14d5cfb3edbe4b0846a404df9002443/src/peripherals/atecc608a-tnglora-se/atca_config.h#L16

> Please note: We have this driver working and tested with Microchip and ST devices.

Reply:
You're right, the title is a bit misleading, I'm sorry. 
Ironically, I am no longer able to reproduce the problem. When I delete my proposed delays, it still works as expected. I don't know why, but back then when I implemented the integration of the secure element, it didn't work without. This could be related to a hardware issue, because I changed the power source in the meantime. 
I am sorry that I didn't check it in more detail before opening a pull request! We can close this request.